manifests/o/Opera/Opera/102.0.4880.16/Opera.Opera.installer.yaml

```diff
@@ -0,0 +1,40 @@
+# Created with Komac v1.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Opera.Opera
+PackageVersion: 102.0.4880.16
+InstallerType: exe
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /allusers=0 /launchbrowser=0 /setdefaultbrowser=0 /enable-installer-stats=0 /enable-stats=0

```

Comment:
It looks like this may be a bug with Komac (cc @russellbanks)

The custom switches and scope seem to not have been preserved from the installer level since they were different for each installer.